File: src/pocketcloud/cloud/command/sender/ConsoleCommandSender.php
<?php

namespace pocketcloud\cloud\command\sender;

use pocketcloud\cloud\terminal\log\CloudLogger;

class ConsoleCommandSender implements ICommandSender {

    public function info(string $message, string ...$params): ICommandSender {
        CloudLogger::get()->info($message, ...$params);
        return $this;
    }

    public function warn(string $message, string ...$params): ICommandSender {
        CloudLogger::get()->warn($message, ...$params);
        return $this;
    }

    public function error(string $message, string ...$params): ICommandSender {
        CloudLogger::get()->error($message, ...$params);
        return $this;
    }

    public function success(string $message, string ...$params): ICommandSender {
        CloudLogger::get()->success($message, ...$params);
        return $this;
    }

    public function debug(string $message, string ...$params): ICommandSender {
        CloudLogger::get()->debug($message, ...$params);
        return $this;
    }
}

File: src/main/java/de/pocketcloud/cloud/console/command/sender/ConsoleCommandSender.java
package de.pocketcloud.cloud.console.command.sender;

import de.pocketcloud.cloud.console.log.CloudLogger;

public final class ConsoleCommandSender implements CommandSender {

    @Override
    public ConsoleCommandSender info(String message, Object... params) {
        CloudLogger.get().info(message, params);
        return this;
    }

    @Override
    public ConsoleCommandSender warn(String message, Object... params) {
        CloudLogger.get().warn(message, params);
        return this;
    }

    @Override
    public ConsoleCommandSender error(String message, Object... params) {
        CloudLogger.get().error(message, params);
        return this;
    }

    @Override
    public ConsoleCommandSender success(String message, Object... params) {
        CloudLogger.get().success(message, params);
        return this;
    }

    @Override
    public ConsoleCommandSender debug(String message, Object... params) {
        CloudLogger.get().debug(message, params);
        return this;
    }
}